tests/test_box_Box.py

```diff
@@ -439,6 +439,67 @@ def test_plot_2d(self):
         box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
         box.plot()
 
+    def test_compute_distances_2d(self):
+        box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
+        points = np.array([[0, 0, 0], [-2.2, -1.3, 0]])
+        query_points = np.array(
+            [[-0.5, -1.3, 0.], [0.5, 0, 0], [-2.2, -1.3, 0.], [0, 0.4, 0]])
+        point_indices = np.array([1, 0, 1, 0])
+        query_point_indices = np.array([0, 1, 2, 3])
+        distances = box.compute_distances(
+            query_points, points, query_point_indices, point_indices)
+        npt.assert_allclose(distances, [0.3, 0.5, 0.0, 0.4], rtol=1e-6)
+
+        # # 1 dimensional array
+        distances = box.compute_distances(
+            query_points, points,
+            query_point_indices[0], point_indices[0])
+        npt.assert_allclose(distances, 0.3, rtol=1e-6)
+
+        with self.assertRaises(IndexError):
+            box.compute_distances(
+                query_points, points, query_point_indices, point_indices + 1)
+        with self.assertRaises(IndexError):
+            box.compute_distances(
+                query_points, points, query_point_indices + 1, point_indices)
+        with self.assertRaises(RuntimeError):
+            box.compute_distances(
+                query_points, points, query_point_indices, point_indices[:-1])
+        with self.assertRaises(RuntimeError):
+            box.compute_distances(
+                query_points, points, query_point_indices[:-1], point_indices)
+
+    def test_compute_distances_3d(self):
+        box = freud.box.Box(2, 3, 4, 1, 0, 0)
+        points = np.array([[0, 0, 0], [-2.2, -1.3, 2]])
+        query_points = np.array(
+            [[-0.5, -1.3, 2.], [0.5, 0, 0], [-2.2, -1.3, 2.], [0, 0, 0.2]])
+        point_indices = np.array([1, 0, 1, 0])
+        query_point_indices = np.array([0, 1, 2, 3])
+        distances = box.compute_distances(
+            query_points, points, query_point_indices, point_indices)
+        npt.assert_allclose(distances, [0.3, 0.5, 0.0, 0.2], rtol=1e-6)
+
+    def test_compute_all_distances_2d(self):
+        box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
+        points = np.array([[0., 0., 0.]])
+        query_points = np.array(
+            [[0.2, 0., 0.], [0., -0.4, 0.], [1., 1., 0.]])
+        distances = box.compute_all_distances(points, query_points)
+        npt.assert_allclose(distances, [[0.2, 0.4, np.sqrt(2)]], rtol=1e-6)
+
+        points = np.array([0., 0., 0.])
+        distances = box.compute_all_distances(points, query_points)
+        npt.assert_allclose(distances, [0.2, 0.4, np.sqrt(2)], rtol=1e-6)
+
+    def test_compute_all_distances_3d(self):
```

Comment:
Be sure one of these tests computes the distance between points across the periodic boundaries.

Reply:
`test_compute_distances_3d` computes distances across periodic boundaries on one of the tilted box sides.

Reply:
One of the distances in `test_compute_all_distances_3d` is across the periodic boundary on a tilted face.